### ISSUE TITLE
rocketeer is not adding through Composer in Laravel 5.5

### ISSUE THREAD

**Issue by rakib-09** [rocketeers/rocketeer#763]
First of all thanks Anahkiasen For this beatiful Project. Recently I am getting this problem when I am trying to require rocketeer in CLI. 
`composer require --dev anahkiasen/rocketeer`

The response is always like this.
![image](https://user-images.githubusercontent.com/11966280/32478276-0852d25e-c3ae-11e7-8b87-4c9e4b7a174b.png)

How Can I solve this ?

**Comment by thearmandov** [comment]
Rocketeer cannot be installed via composer on laravel 5.5.

**Comment by JanKyselica** [comment]
Same problem on L5.4.36
```
Problem 1
    - Conclusion: remove laravel/framework v5.4.36
    - Conclusion: don't install laravel/framework v5.4.36
    - anahkiasen/rocketeer 2.2.0 requires illuminate/log ~4.2 -> satisfiable by illuminate/log[v4.2.1, v4.2.12, v4.2.16, v4.2.17, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - anahkiasen/rocketeer 2.2.1 requires illuminate/log ~4.2 -> satisfiable by illuminate/log[v4.2.1, v4.2.12, v4.2.16, v4.2.17, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - anahkiasen/rocketeer 2.2.2 requires illuminate/log ~4.2 -> satisfiable by illuminate/log[v4.2.1, v4.2.12, v4.2.16, v4.2.17, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - anahkiasen/rocketeer 2.2.3 requires illuminate/log ~4.2 -> satisfiable by illuminate/log[v4.2.1, v4.2.12, v4.2.16, v4.2.17, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - anahkiasen/rocketeer 2.2.4 requires illuminate/log ~4.2 -> satisfiable by illuminate/log[v4.2.1, v4.2.12, v4.2.16, v4.2.17, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - anahkiasen/rocketeer 2.2.5 requires illuminate/log ~4.2 -> satisfiable by illuminate/log[v4.2.1, v4.2.12, v4.2.16, v4.2.17, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9].
    - don't install illuminate/log v4.2.1|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.12|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.16|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.17|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.2|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.3|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.4|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.5|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.6|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.7|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.8|don't install laravel/framework v5.4.36
    - don't install illuminate/log v4.2.9|don't install laravel/framework v5.4.36
    - Installation request for laravel/framework (locked at v5.4.36, required as 5.4.*) -> satisfiable by laravel/framework[v5.4.36].
    - Installation request for anahkiasen/rocketeer ^2.2 -> satisfiable by anahkiasen/rocketeer[2.2.0, 2.2.1, 2.2.2, 2.2.3, 2.2.4, 2.2.5].
```

**Comment by KiberMath** [comment]
may be this plugin can help? https://github.com/bamarni/composer-bin-plugin
i've just noticed it.

**Comment by Ciaro** [comment]
Hi @KiberMath 
Thanks for the tip! I successfully implemented rocketeer in Laravel 5.5 by using the composer plugin you mentioned.

**Comment by Ciaro** [comment]
Keep in mind that this is a work-around and the actual linking to the Laravel framework still does not work.
